### PR TITLE
Update unattended to 4.3

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
||

## Description

This PR updates the unattended script in branch 4.3 with the latest changes. 

## Tests

- Build the package in any supported platform
  - [ ] Linux
  - [ ] Windows
  - [ ] macOS
  - [ ] Solaris
  - [ ] AIX
  - [ ] HP-UX
- [ ] Package installation
- [ ] Package upgrade
- [ ] Package downgrade
- [ ] Package remove
- [ ] Package install/remove/install
- [ ] Change added to CHANGELOG.md

- Tests for Linux RPM
  - [ ] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [ ] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] Package install/remove/install
  - [ ] Package install/purge/install
  - [ ] Check file permissions after installing the package
- Tests for macOS
  - [ ] Test the package from macOS Sierra to Mojave
- Tests for Solaris
  - [ ] Test the package on Solaris 10
  - [ ] Test the package on Solaris 11
  - [ ] Check file permissions on Solaris 11 template
- Tests for IBM AIX
  - [ ] `%files` section is correctly updated if necessary
  - [ ] Check the changes from IBM AIX 5 to 7

### CODE REVIEW

File: unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh [hunk 1]
```diff
@@ -261,8 +267,8 @@ installElasticsearch() {
 
         logger "Configuring Elasticsearch..."
 
-        eval "curl -so /etc/elasticsearch/elasticsearch.yml https://packages.wazuh.com/resources/4.2/elastic-stack/elasticsearch/7.x/elasticsearch_all_in_one.yml --max-time 300 $debug"
-        eval "curl -so /usr/share/elasticsearch/instances.yml https://packages.wazuh.com/resources/4.2/elastic-stack/instances_aio.yml --max-time 300 $debug"
+        eval "curl -so /etc/elasticsearch/elasticsearch.yml https://packages.wazuh.com/resources/4.3/elastic-stack/elasticsearch/7.x/elasticsearch_all_in_one.yml --max-time 300 $debug"
+        eval "curl -so /usr/share/elasticsearch/instances.yml https://packages.wazuh.com/resources/4.3/elastic-stack/instances_aio.yml --max-time 300 $debug"
```
Comment (rauldpm) [inline unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh:271]: This shouldn't be hard-coded.

File: unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh [hunk 2]
```diff
@@ -317,21 +323,21 @@ installFilebeat() {
...
     fi
     if [  "$?" != 0  ]
     then
         logger -e "Filebeat installation failed"
         exit 1;
     else
-        eval "curl -so /etc/filebeat/filebeat.yml https://packages.wazuh.com/resources/4.2/elastic-stack/filebeat/7.x/filebeat_all_in_one.yml --max-time 300  $debug"
-        eval "curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/4.2/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 $debug"
+        eval "curl -so /etc/filebeat/filebeat.yml https://packages.wazuh.com/resources/4.3/elastic-stack/filebeat/7.x/filebeat_all_in_one.yml --max-time 300  $debug"
+        eval "curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/4.3/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 $debug"
```
Comment (rauldpm) [inline unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh:340]: Same

File: unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh [hunk 3]
```diff
@@ -354,24 +360,24 @@ installKibana() {
...
-        eval "apt-get install kibana=7.11.2 -y -q  $debug"
+        eval "apt-get install kibana=${ELK_VER} -y -q  $debug"
     fi
     if [  "$?" != 0  ]
     then
         logger -e "Kibana installation failed"
         exit 1;
     else
-        eval "curl -so /etc/kibana/kibana.yml https://packages.wazuh.com/resources/4.2/elastic-stack/kibana/7.x/kibana_all_in_one.yml --max-time 300 $debug"
+        eval "curl -so /etc/kibana/kibana.yml https://packages.wazuh.com/resources/4.3/elastic-stack/kibana/7.x/kibana_all_in_one.yml --max-time 300 $debug"
```
Comment (rauldpm) [inline unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh:376]: Same

File: unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh [hunk 4]
```diff
@@ -245,7 +252,7 @@ installElasticsearch() {
         logger "Done"
 
         logger "Configuring Elasticsearch..."
-        eval "curl -so /etc/elasticsearch/elasticsearch.yml https://packages.wazuh.com/resources/4.2/elastic-stack/unattended-installation/distributed/templates/elasticsearch_unattended.yml --max-time 300 $debug"
+        eval "curl -so /etc/elasticsearch/elasticsearch.yml https://packages.wazuh.com/resources/4.3/elastic-stack/unattended-installation/distributed/templates/elasticsearch_unattended.yml --max-time 300 $debug"
```
Comment (rauldpm) [inline unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh:255]: Same

File: unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh [hunk 5]
```diff
@@ -434,25 +442,25 @@ installKibana() {
...
+        eval "apt-get install kibana=${ELK_VER} -y -q  $debug"
     fi
     if [  "$?" != 0  ]
     then
         logger -e "Kibana installation failed"
         exit 1;
     else
         disableRepos
-        eval "curl -so /etc/kibana/kibana.yml https://packages.wazuh.com/resources/4.2/elastic-stack/unattended-installation/distributed/templates/kibana_unattended.yml --max-time 300 $debug"
+        eval "curl -so /etc/kibana/kibana.yml https://packages.wazuh.com/resources/4.3/elastic-stack/unattended-installation/distributed/templates/kibana_unattended.yml --max-time 300 $debug"
```
Comment (rauldpm) [inline unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh:459]: Same

File: unattended_scripts/elastic-stack/unattended-installation/distributed/wazuh-server-installation.sh [hunk 6]
```diff
@@ -283,8 +283,8 @@ installFilebeat() {
         logger -e "Filebeat installation failed"
         exit 1;
     else
-        eval "curl -so /etc/filebeat/filebeat.yml https://packages.wazuh.com/resources/4.2/elastic-stack/unattended-installation/distributed/templates/filebeat.yml --max-time 300  $debug"
-        eval "curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/4.2/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 $debug"
+        eval "curl -so /etc/filebeat/filebeat.yml https://packages.wazuh.com/resources/4.3/elastic-stack/unattended-installation/distributed/templates/filebeat.yml --max-time 300  $debug"
```
Comment (rauldpm) [inline unattended_scripts/elastic-stack/unattended-installation/distributed/wazuh-server-installation.sh:286]: Same

File: unattended_scripts/open-distro/unattended-installation/distributed/wazuh-server-installation.sh [hunk 7]
```diff
@@ -211,8 +211,8 @@ installFilebeat() {
         logger -e "Filebeat installation failed"
         exit 1;
     else
-        eval "curl -so /etc/filebeat/filebeat.yml https://packages.wazuh.com/resources/4.2/open-distro/unattended-installation/distributed/templates/filebeat.yml --max-time 300 ${debug}"
-        eval "curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/4.2/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 ${debug}"
+        eval "curl -so /etc/filebeat/filebeat.yml https://packages.wazuh.com/resources/4.3/open-distro/unattended-installation/distributed/templates/filebeat.yml --max-time 300 ${debug}"
+        eval "curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/4.3/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 ${debug}"
```
Comment (rauldpm) [inline unattended_scripts/open-distro/unattended-installation/distributed/wazuh-server-installation.sh:215]: Same

File: unattended_scripts/open-distro/unattended-installation/unattended-installation.sh [hunk 8]
```diff
@@ -351,7 +351,7 @@ installFilebeat() {
     else
         filebeatinstalled="1"
         eval "curl -so /etc/filebeat/filebeat.yml ${resources}/open-distro/filebeat/7.x/filebeat_unattended.yml --max-time 300  ${debug}"
-        eval "curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/4.0/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 ${debug}"
+        eval "curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/4.3/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 ${debug}"
```
Comment (rauldpm) [inline unattended_scripts/open-distro/unattended-installation/unattended-installation.sh:354]: Same

**Review by rauldpm — CHANGES_REQUESTED** [review]
Minor change. Change hardcoding if possibble by ${WAZUH_MAJOR}

File: unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh (hunk 1 above)
Comment (DFolchA) [reply unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh]: Done in 8a5379c

File: unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh (hunk 2 above)
Comment (DFolchA) [reply unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh]: Done in 8a5379c

File: unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh (hunk 3 above)
Comment (DFolchA) [reply unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh]: Done in 8a5379c

File: unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh (hunk 4 above)
Comment (DFolchA) [reply unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh]: Done in 8a5379c

File: unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh (hunk 5 above)
Comment (DFolchA) [reply unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh]: Done in 8a5379c

File: unattended_scripts/elastic-stack/unattended-installation/distributed/wazuh-server-installation.sh (hunk 6 above)
Comment (DFolchA) [reply unattended_scripts/elastic-stack/unattended-installation/distributed/wazuh-server-installation.sh]: Done in 8a5379c

File: unattended_scripts/open-distro/unattended-installation/distributed/wazuh-server-installation.sh (hunk 7 above)
Comment (DFolchA) [reply unattended_scripts/open-distro/unattended-installation/distributed/wazuh-server-installation.sh:215]: Done in 8a5379c

File: unattended_scripts/open-distro/unattended-installation/unattended-installation.sh (hunk 8 above)
Comment (DFolchA) [reply unattended_scripts/open-distro/unattended-installation/unattended-installation.sh]: Done in 8a5379c

**Review by rauldpm — APPROVED** [review]
LGTM

File: unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh [hunk 9]
```diff
@@ -165,15 +166,16 @@ readFileUsers() {
...
-        User:
-        name: wazuh
-        password: wazuhpasword
-        User:
-        name: kibanaserver
-        password: kibanaserverpassword"
-        exit 1
+	logger -e "The password file doesn't have a correct format.
+
+It must have this format:
```
Comment (alberpilot) [inline unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh:171]: Put the output on a variable for better formatting and give the variable to logger

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

File: unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh (hunk 9 above)
Comment (DFolchA) [reply unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh]: Done in a7004d7

**Review by alberpilot — APPROVED** [review]
LGTM

**Pull request merged**